lms/templates/courseware/static_tab.html

```diff
@@ -1,25 +1,33 @@
+## mako
+
+<%page expression_filter="h"/>
+<%!
+from openedx.core.djangolib.markup import HTML
+%>
+
 <%inherit file="/main.html" />
 <%block name="bodyclass">view-in-course view-statictab ${course.css_class or ''}</%block>
 <%namespace name='static' file='/static_content.html'/>
 
 <%block name="headextra">
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
+${HTML(fragment.head_html())}
 </%block>
 <%block name="js_extra">
```

Comment:
Is `js_extra` always added to the bottom?  Maybe it would make more sense to rename this block to `footer_extra`, which happens to have JS in it, than the other way around?  I might be missing something here though.